app/controllers/search.js

```diff
@@ -1,9 +1,18 @@
 import Controller from '@ember/controller';
 
 export default Controller.extend({
-  queryParams: {
-    query: 'q'
-  },
+  queryParams: ['page', { query: 'q' }],
 
-  query: ''
+  page: 1,
+  query: '',
+
+  actions: {
+    prevPage() {
+      this.set('page', this.page - 1);
```

Comment:
Can you click on a page number directly? In that case it seems like both of these actions should just be replaced with `setPage` and the computation can be done in the component itself.

Reply:
👍